manuals/build_manuals.py

```
@@ -0,0 +1,279 @@
+
+OUTPUT_DIR = WEBSITE_REPO / "assets" / "manuals"
+STYLE_TEX = MANUALS_DIR / "style.tex"
+
+
+# ---------- tooling ----------
+
+def require_tool(name: str) -> str:
+    path = shutil.which(name)
+    if not path:
+        print(f"error: required tool not found on PATH: {name}", file=sys.stderr)
+        print(f"       install hints:", file=sys.stderr)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)